recipe/build-lib.sh

```diff
@@ -8,7 +28,7 @@ fi
 # Build vanilla version (no avx)
 ./configure --prefix=${PREFIX} --exec-prefix=${PREFIX} \
   --with-blas=-lblas --with-lapack=-llapack \
-  ${CUDA_CONFIG_ARG}
+  ${CUDA_CONFIG_ARG} --with-cuda-arch="${CUDA_ARCH}" || exit 1
```

Comment:
Would it make sense to add this to the `CUDA_CONFIG_ARG` above?

Reply:
I tried this first of course, but couldn't get the quotes to get passed through correctly, and then the compilation would fail.